### ISSUE TITLE
IndexError raised by _ftp_parse.decode_windowsnt() in v2.4.13

### ISSUE THREAD

**Issue by RJPercival** [PyFilesystem/pyfilesystem2#478]
Since updating from v2.4.11 to v2.4.13, we've been unable to access FTP servers due to the following code raising `IndexError("no such group")`:
https://github.com/PyFilesystem/pyfilesystem2/blob/ac8a91add9259c5be2caf1746be1db146ea78390/fs/_ftp_parse.py#L195-L197

For help with debugging, this was the value of `raw_info`:
```python
{
  basic: {
    is_dir: True, 
    name: 'validation_warnings'
  }, 
  details: {
    type: 1
  }, 
  ftp: {
    ls: '04-13-21  08:21AM       <DIR>          validation_warnings'
  }
}
```

**Comment by lurch** [comment]
ping @atollk could this be related to #439 ?

**Comment by RJPercival** [comment]
Ah I've just discovered that we were monkey-patching `RE_WINDOWSNT` to fix a bug (not handling AM/PM) that is fixed in v2.4.13. If we remove our monkey-patch, that should fix this issue. Thanks anyway!
